cmd/history.go

```diff
@@ -81,24 +81,24 @@ func ExecuteHistoryCleanupCommand(args []string) error {
 	}
 	var matchingTags = cleanup.GetMatchingTags(&gitCandidates, &imageStreamTags, matchOption)
 
-	activeImageStreamTags, err := openshift.GetActiveImageStreamTags(namespace, image, matchingTags)
+	activeImageStreamTags, err := openshift.GetActiveImageStreamTags(namespace, imageName, matchingTags)
 	if err != nil {
-		return fmt.Errorf("could not retrieve active image stream tags for '%s/%s': %w", namespace, image, err)
+		return fmt.Errorf("could not retrieve active image stream tags for '%s/%s': %w", namespace, imageName, err)
 	}
 
 	inactiveTags := cleanup.GetInactiveImageTags(&activeImageStreamTags, &matchingTags)
 	inactiveTags = cleanup.LimitTags(&inactiveTags, c.Keep)
 	if len(inactiveTags) == 0 {
 		log.WithFields(log.Fields{
 			"\n - namespace": namespace,
-			"\n - imageName": image,
+			"\n - 📺 image":   imageName,
 		}).Info("No inactive image stream tags found")
```

Comment:
Something went wrong with the formatting here. can you change that to:
```
log.WithFields(log.Fields{
  "namespace": namespace,
  "imageName": image,
  "image": imageName,
})
```

Reply:
We now use a uniform layout and naming in the the output. This was the only occurrence that used `"imageName"`, IIRC. We should probably keep it.

The unicode icons is, technically, a hack to influence the the sort order; it's a compromise: @ZuGao and I didn't find a sensible way to influence the sort order or make the log output otherwise more readable; it adds some fun to the CLI, that's probably not bad. In the long run we need to review the log output and maybe refactor it (the log library we currently use is not meant for humans).

The formatting only _appears_ weird, because the unicode symbols have a different width than the usual characters. `go fmt` passes, though, without changing the source code any further.

Reply:
Well, the logging library currently used with the fields etc. is more tailored towards logging infrastructure, e.g. ElasticSearch to ease on the metadata scraping for better filtering. I was under impression that the primary execution environments would be CI pipelines and other sorts of automated jobs, e.g. cron jobs, and less for humans overall. Icons wouldn't make sense then.

It appears we need to make a clear architecture decision regarding logging and then possibly overhaul the complete logging if decision goes in favor to "humans". In a different PR/issue though.

Reply:
It is even written in the seiso Readme ;)